cdap-docs/faqs/source/cloudera-manager.rst

```diff
@@ -0,0 +1,118 @@
+using `Cloudera Manager <http://www.cloudera.com/content/www/en-us/products/cloudera-manager.html>`__. 
+
+
+When I run a query, I see "Permission Error" in the logs. What do I do?
+-----------------------------------------------------------------------
+Some versions of Hive may try to create a temporary staging directory at the table
+location when executing queries. If you are seeing permissions errors when running a
+query, try setting ``hive.exec.stagingdir`` in your Hive configuration to
+``/tmp/hive-staging``. 
+
+This can be done in Cloudera Manager using the *Hive Client
+Advanced Configuration Snippet (Safety Valve) for hive-site.xml* configuration field.
+
+
+I'm getting a message about "Missing System Artifacts". How do I fix that?
+--------------------------------------------------------------------------
+The bundled system artifacts are included in the CDAP parcel, located in a subdirectory
+of Cloudera Manager's ``${PARCELS_ROOT}`` directory, for example::
+
+  /opt/cloudera/parcels/CDAP/master/artifacts
+
+Ensure that the ``App Artifact Dir`` configuration option points to this path on disk. Since this
+directory can change when CDAP parcels are upgraded, users are encouraged to place
+these artifacts in a static directory outside the parcel root, and configure accordingly.
+
+
+.. _faqs-cloudera-direct-parcel-access:
+
+I'd like to directly access the parcels. How do I do that?
+----------------------------------------------------------
+If you need to download and install the parcels directly (perhaps for a cluster that does
+not have direct network access), the parcels are available by their full URLs. As they are
+stored in a directory that does not offer browsing, they are listed here:
+
+.. parsed-literal::
+  |http:|//repository.cask.co/parcels/cdap/latest/CDAP-|version|-1-el6.parcel
+  |http:|//repository.cask.co/parcels/cdap/latest/CDAP-|version|-1-precise.parcel
+  |http:|//repository.cask.co/parcels/cdap/latest/CDAP-|version|-1-trusty.parcel
+  |http:|//repository.cask.co/parcels/cdap/latest/CDAP-|version|-1-wheezy.parcel
+  
+If you are hosting your own internal parcel repository, you may also want the
+``manifest.json``:
+
+.. parsed-literal::
+  |http:|//repository.cask.co/parcels/cdap/latest/manifest.json
+
+The ``manifest.json`` can always be referred to for the list of latest available parcels.
+
+Previously released parcels can also be accessed from their version-specific URLs.  For example:
+
+.. parsed-literal::
+  |http:|//repository.cask.co/parcels/cdap/2.8/CDAP-2.8.0-1-el6.parcel
+  |http:|//repository.cask.co/parcels/cdap/2.8/CDAP-2.8.0-1-precise.parcel
+  |http:|//repository.cask.co/parcels/cdap/2.8/CDAP-2.8.0-1-trusty.parcel
+  |http:|//repository.cask.co/parcels/cdap/2.8/CDAP-2.8.0-1-wheezy.parcel
+  
+
+.. _faqs-cloudera-troubleshooting-upgrade-cdh:
+
+I missed doing a step while upgrading; how do I fix my system?
```

Comment:
This is not really cloudera-manager specific.  We should probably move it to the CDAP FAQ next to the other upgrade-related question(s)